cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -11,12 +11,129 @@ Docker Image
 
 
 Docker is one of the easiest ways to start working with CDAP without having to manually
-configure anything. A Docker image with the CDAP SDK pre-installed is available on the Docker Hub
+configure anything. A Docker container with the CDAP SDK pre-installed is available on the Docker Hub
 for download.
 
-To use the **Docker image**, you can either use Docker's :ref:`Kitematic
-<docker-kitematic>` (on Mac OS X and Windows) |---| a graphical user interface for running
-Docker containers |---| or start the container from :ref:`a command line <docker-command-line>`.
+To use the **Docker image**, you can either start the container from :ref:`a command line
+<docker-command-line>` or use Docker's :ref:`Kitematic <docker-kitematic>` (on Mac OS X
+and Windows), a graphical user interface for running Docker containers.
+
+.. _docker-command-line:
+
+Docker from a Command Line
+==========================
+
+Docker environments are available for a variety of platforms. Download and install Docker for your platform by
+following the `platform-specific installation instructions <https://docs.docker.com/engine/installation/>`__
+from `Docker.com <https://docker.com>`__, and verify that the Docker environment is working and has
+started correctly.
+  
+#. If you are not running on Linux, you will need to create and start the Docker Virtual Machine (VM) before you
+   can use containers. For example:
+   
+   .. tabbed-parsed-literal::
+     :tabs: "Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+     
+     $ docker-machine create --driver virtualbox cdap
+     $ docker-machine env cdap
+     
+   This will create a new Docker virtual machine named ``cdap`` and print out the environment.
```

Comment:
I like `This will create a new virtual machine using VirtualBox for Docker named cdap and print out...` but I'm not married to it. What you have is fine, really.